Document the extra_groups and owner_extra_groups options (#999)
ubuntu/authd: docs/howto/configure-authd.md

owner = ""
```

(ref::config-user-groups)=
## Configure user groups

> Some brokers support adding users to groups configured in the identity provider.

Reviewer: ```suggestion
Some brokers support adding users to groups that are configured in the identity provider.
```

I think the "that are" here helps to break up the sentence, which otherwise has a series of technical terms ("users", "groups", "configured").

> See the [group management guide](reference::group-management) for more details.

> In addition, you can configure extra groups that authd users are automatically
> added to upon login by specifying them in the `users` section of the broker
> configuration file:

Reviewer: ```suggestion
In addition, you can configure extra groups for authd users.
On login, the users are added to these groups automatically.
Specify any extra groups in the `users` section of the broker
configuration file:
```

Original sentence was somewhat complex, so I divided into smaller sentences.
This also helps avoid "[...] added to upon [...]", which is slightly cumbersome.

```ini
[users]
## A comma-separated list of local groups which authd users will be
## added to upon login.
## Example: extra_groups = users
#extra_groups =
```

> There is also an `owner_extra_groups` option for specifying additional local groups
> to which only the owner user is added (see the
> [allowed users section](#configure-allowed-users) for details on the owner role):

Reviewer: ```suggestion
There is also an `owner_extra_groups` option for specifying additional local groups,
to which only the user with the [owner role](#configure-allowed-users) is added:
```

* I think "user with the owner role" is more clear than "owner user"
* It's OK here to associate the hyperlink with the term someone is most likely to query, and then we can remove the extra line
* The colon makes more sense after the sentence that specifies the configuration rather than the aside
